File: Bars.GkhDI/Entities/Service/WorkRepairList.cs
namespace Bars.GkhDi.Entities
{
    using System;
    using Bars.Gkh.Entities;

    /// <summary>
    /// ППР список работ
    /// </summary>
    public class WorkRepairList : BaseGkhEntity
    {
        /// <summary>
        /// Услуга
        /// </summary>
        public virtual BaseService BaseService { get; set; }

        /// <summary>
        /// Группа работ по ППР
        /// </summary>
        public virtual GroupWorkPpr GroupWorkPpr { get; set; }

        /// <summary>
        /// Запланированный объем
        /// </summary>
        public virtual decimal? PlannedVolume { get; set; }

        /// <summary>
        /// Запланированная стоимость
        /// </summary>
        public virtual decimal? PlannedCost { get; set; }

        /// <summary>
        /// Фактический объем
        /// </summary>
        public virtual decimal? FactVolume { get; set; }

        /// <summary>
        /// Фактический стоимость
        /// </summary>
        public virtual decimal? FactCost { get; set; }

        /// <summary>
        /// Дата начала
        /// </summary>
        public virtual DateTime? DateStart { get; set; }

        /// <summary>
        /// Дата окончания
        /// </summary>
        public virtual DateTime? DateEnd { get; set; }
    }
}

File: Bars.GkhDI/libs/B4/model/Service/WorkRepairList.js
Ext.define('B4.model.service.WorkRepairList', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'WorkRepairList'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'BaseService', defaultValue: null },
        { name: 'GroupWorkPpr', defaultValue: null },
        { name: 'GroupWorkPprId', defaultValue: null },
        { name: 'Name' },
        { name: 'PlannedVolume', defaultValue: null },
        { name: 'PlannedCost', defaultValue: null },
        { name: 'FactVolume', defaultValue: null },
        { name: 'FactCost', defaultValue: null },
        { name: 'DateStart', defaultValue: null },
        { name: 'DateEnd', defaultValue: null }
    ]
});